qemu/tests/usb_host.py

```diff
@@ -108,30 +147,39 @@ def usb_dev_unplug(dev):
     match_add = "New USB device found, "
     match_add += "idVendor=%s, idProduct=%s" % (vendorid, productid)
     match_del = "USB disconnect"
+    usb_stick = "Mass Storage" in process.getoutput(lsusb_cmd)
 
     error_context.context("Log into guest", logging.info)
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     session = vm.wait_for_login()
 
-    usb_dev_verify()
-    usb_devs = get_usb_host_dev()
-    for dev in usb_devs:
-        usb_dev_unplug(dev)
-
-    repeat_times = int(params.get("usb_repeat_times", "1"))
-    for i in range(repeat_times):
-        msg = "Hotplug (iteration %d)" % (i+1)
-        usb_params["id"] = "usbhostdev%s" % i
-        if params.get("usb_check_isobufs", "no") == "yes":
-            # The value of isobufs could only be in '4, 8, 16'
-            isobufs = (2 << (i % 3 + 1))
-            usb_params["isobufs"] = isobufs
-            msg += ", with 'isobufs' option set to %d." % isobufs
-        error_context.context(msg, logging.info)
-        usb_dev = qdevices.QDevice("usb-host", usb_params)
-        usb_dev_hotplug(usb_dev)
+    try:
         usb_dev_verify()
-        usb_dev_unplug(usb_dev)
-
-    session.close()
+        if usb_stick:
+            iozone_test = generate_instance(params, vm, 'iozone')
```

Comment:
Here, iozone will be prepared in vm.   However, it should be redundant if usb stick mount failed.
How about check mount status first, then prepare iozone, and do stick io ?

Reply:
Yes, you're right! Updated, thanks!